WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -224,25 +224,10 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 
 		try
 		{
-			try
-			{
-				using CancellationTokenSource timeUntilOutputRegCts = new(timeUntilOutputReg + ExtraPhaseTimeoutMargin);
-				using CancellationTokenSource linkedCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, timeUntilOutputRegCts.Token);
+			using CancellationTokenSource timeUntilOutputRegCts = new(timeUntilOutputReg + ExtraPhaseTimeoutMargin);
+			using CancellationTokenSource linkedCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, timeUntilOutputRegCts.Token);
 
-				registeredAliceClientAndCircuits = await ProceedWithInputRegAndConfirmAsync(smartCoins, roundState, linkedCts.Token).ConfigureAwait(false);
-			}
-			catch (UnexpectedRoundPhaseException ex)
```

Comment:
If you put this to the end, it won't have the same behavior as before. The client should be indulgent only in input registration phase regarding the exception of UnexpectedRoundPhaseException.